lib/rubocop/cop/lint/return_in_void_context.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for the use of a return with a value in a context
+      # where it the value will be ignored. (initialize and setter methods)
+      #
+      # @example
+      #
+      #   # bad
```

Comment:
You've indented the bad example by 3 spaces instead of 2 and this breaks the generated documentation.